### DISCUSSION TITLE
Is it possible to use AG with a remote PC that's NOT on the same LAN?

### DISCUSSION THREAD

**Discussion by ad2forum** [apohl79/audiogridder#425] [Help]
Hi! I have a MBP and a remote Win PC server, that is physically located in the another country.
I've installed AG Client on the MBP and AG Server on Win PC server. All seems to work fine on itself.
But can't connect to the server no matter how or what I do! VNC, RDP, disabling firewalls etc.
Is it even possible with this setup?
Thanks.

**Comment by apohl79** [comment]
Currently, you need the server to be able to connect to the client, a bit like classic FTP works. To verify this, you need to be able to ping the server from your MPB as well as the MPB from the server. This is likely your problem.

Other than that it should work, but the long distance and latency might cause issues.

**Reply by ad2forum** [reply]
Thanks for such a quick response! I can ping my WinServer from MBP no problem, but vice versa it won't work, since I don't have a dedicated IP from my provider. So I guess that's the issue here.
Maybe there's a workaround to this?

**Reply by apohl79** [reply]
Unless you manage to get the server to connect to the client, there is no workaround. You will probably have to wait for the first 1.2 beta version (no ETA I'm sorry), which will not require the server to connect to the client. The client will establish all connections then.
